src/main/starlark/builtins_bzl/common/cc/cc_helper.bzl

```diff
@@ -944,16 +944,10 @@ def _map_to_list(m):
         result.append((k, v))
     return result
 
-# Returns a list of (Artifact, Label) tuples. Each tuple represents an input source
-# file and the label of the rule that generates it (or the label of the source file itself if it
-# is an input file).
-def _get_srcs(ctx):
-    if not hasattr(ctx.attr, "srcs"):
-        return []
-
-    # "srcs" attribute is a LABEL_LIST in cc_rules, which might also contain files.
+# "srcs" attribute is a LABEL_LIST in cc_rules, which might also contain files.
+def _calculate_artifact_label_map(srcs, attr):
```

Comment:
nit: rename `srcs` -> `attr_list`, `attr` -> `attr_name`, `src` -> `attr`. (The code is not used only for srcs attribute anymore)

change the comment over the function into a doc string:
```
"""
Converts a label_list attribute into a list of (Artifact, Label) tuples.

Each tuple represents an input source file and the label of the rule that generates it (or the label of the source file itself if it is an input file.
"""
```
Remove other (duplicated) comments above _get_srcs and _get_cpp20module_interfaces.